### PR TITLE
Lesson 9 assignment 8, Object Oriented Mailroom, for review.

### PR DESCRIPTION
**Pull request by Brent-K**

### CODE REVIEW

File: students/brent_kieszling/Lesson_09/Assignment_08/donor_modles.py [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+        return self._profiles
+
+    def find_donor(self, person):
+        a = False
+        for item in self._profiles:
+            if item.person == person:
+                a = True
+                return item
+        if a == False:
+            raise IndexError
```
Comment (natasha-aleksandrova) [inline students/brent_kieszling/Lesson_09/Assignment_08/donor_modles.py:152]: you can simplify this code a bit
```
for item in self._profiles:
    if item.person == person:
        return item
else:
    raise IndexError
```

**Review by natasha-aleksandrova — APPROVED** [review]
nice work!

**Comment by Brent-K** [comment]
Thanks for the code recommendation, your way is much cleaner.

**Comment by natasha-aleksandrova** [comment]
of course, you're welcome!

**Pull request merged**